draft-ietf-rtcweb-jsep.xml

```diff
@@ -1477,7 +1477,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
           <t>This API changes the local media state; among other
           things, it sets up local resources for sending and encoding
-          media.</t>
+          media.
+          A rollback will revert to the current local description.</t>
```

Comment:
'Current remote description'. But not sure this is adding a lot here, may be simpler to just remove.